crates/wash-cli/src/wit.rs

```diff
@@ -66,6 +71,19 @@ pub struct FetchArgs {
     pub common: CommonPackageArgs,
 }
 
+#[derive(Args, Debug, Clone)]
+pub struct PublishArgs {
+    /// The file to publish
+    file: PathBuf,
+
+    /// The registry domain to use. Overrides configuration file(s).
+    #[arg(long = "registry", value_name = "REGISTRY", env = "WKG_REGISTRY")]
```

Comment:
Should this be `WASH_REGISTRY`?

Feels a little strange to have `WKG_` variables in `wash` imo.

Reply:
Is there a need to differentiate between the registry for `wash push`? I don't think that has an env var, but maybe `WASH_WKG_REGISTRY`? Does `wkg` itself use the `WKG_REGISTRY` variable?

Reply:
This is for parity sake with `wkg` so people don't have to have two different env vars to set. I did something similar with the common package opts

Reply:
Oh @lachieh's comment wasn't showing up when I wrote my response. He's correct

Reply:
My vote is for `--wit-registry` & `WASH_WIT_REGISTRY`